src/components/HamburgerToggle/HamburgerToggle.module.scss

```diff
@@ -0,0 +1,73 @@
+
+
+.hamOverlay{
+    position: absolute;
+    background-color: #0D4E67;
+    border-radius: 4px;
+    width: 100%;
+    height: 100%;
+
+    cursor: pointer;
+}
+
+$wrapperHOffset: 10%;
+$wrapperVOffset: 20%;
+.hamWrapper{
+    position: absolute;
+    top: $wrapperVOffset;
+    bottom: $wrapperVOffset;
+    left: $wrapperHOffset;
+    right: $wrapperHOffset;
+
+    & span{
```

Comment:
I'm a little confused why do you add & sign? [stack overflow](https://stackoverflow.com/questions/38804099/scss-and-to-what-it-refers-to/38804416). From my understanding adding `& span{}` like this doesn't do anything. While adding `&span{}` would mean `.hamWrapperspan{}`

Reply:
No I did not mean `.hamWrapperspan{}`. span is for white horizontal lines and I need to access `.hamWrapper` 's child which is span. If you understand what I am saying.

Reply:
Yes I understand your scss need to affect only span elements that has relationship with class hamWrapper.